src/main/fc/rc_modes.h

```diff
@@ -26,61 +26,61 @@
 #define BOXID_NONE 255
 
 typedef enum {
-    BOXARM           = 0,
-    BOXANGLE         = 1,
-    BOXHORIZON       = 2,
-    BOXNAVALTHOLD    = 3,    // old BOXBARO
-    BOXHEADINGHOLD   = 4,    // old MAG
-    BOXHEADFREE      = 5,
-    BOXHEADADJ       = 6,
-    BOXCAMSTAB       = 7,
-    BOXNAVRTH        = 8,    // old GPSHOME
-    BOXNAVPOSHOLD    = 9,    // old GPSHOLD
-    BOXMANUAL        = 10,
-    BOXBEEPERON      = 11,
-    BOXLEDLOW        = 12,
-    BOXLIGHTS        = 13,
-    BOXNAVLAUNCH     = 14,
-    BOXOSD           = 15,
-    BOXTELEMETRY     = 16,
-    BOXBLACKBOX      = 17,
-    BOXFAILSAFE      = 18,
-    BOXNAVWP         = 19,
-    BOXAIRMODE       = 20,
-    BOXHOMERESET     = 21,
-    BOXGCSNAV        = 22,
-    BOXSURFACE       = 24,
-    BOXFLAPERON      = 25,
-    BOXTURNASSIST    = 26,
-    BOXAUTOTRIM      = 27,
-    BOXAUTOTUNE      = 28,
-    BOXCAMERA1       = 29,
-    BOXCAMERA2       = 30,
-    BOXCAMERA3       = 31,
-    BOXOSDALT1       = 32,
-    BOXOSDALT2       = 33,
-    BOXOSDALT3       = 34,
-    BOXNAVCOURSEHOLD = 35,
-    BOXBRAKING       = 36,
-    BOXUSER1         = 37,
-    BOXUSER2         = 38,
-    BOXFPVANGLEMIX   = 39,
-    BOXLOITERDIRCHN  = 40,
-    BOXMSPRCOVERRIDE = 41,
-    BOXPREARM        = 42,
-    BOXTURTLE        = 43,
-    BOXNAVCRUISE     = 44,
-    BOXAUTOLEVEL     = 45,
-    BOXPLANWPMISSION = 46,
-    BOXSOARING       = 47,
-    BOXUSER3         = 48,
-    BOXUSER4         = 49,
-    BOXCHANGEMISSION = 50,
-    BOXBEEPERMUTE    = 51,
-    BOXMULTIFUNCTION = 52,
-    BOXMIXERPROFILE      = 53,
-    BOXMIXERTRANSITION   = 54,
-    BOXANGLEHOLD     = 55,
+    BOXARM,
+    BOXANGLE,
+    BOXHORIZON,
```

Comment:
Having the explicit numbers for the enumeration is nice for referencing the code to know what each of the modes are.

Reply:
I agree with that, more useful than counting

Reply:
I guess it's useful for debugging, which will only return the number, in which case commented numbering make more sense, let the enum apply the sequential numbering itself surely.